packages/ui/Skinnable/util.js

```diff
@@ -26,7 +27,58 @@ const objectify = (arg) => {
 
 const preferDefined = (a, b) => ((a != null) ? a : b);
 
+function determineSkin (defaultSkin, authorSkin, parentSkin) {
+	return authorSkin || defaultSkin || parentSkin;
```

Comment:
`defaultSkin` here is the local default, not the app default.  I'm not sure if there's a way to clarify that or if it needs to, but I was confused at first.

Reply:
Adding some docs for this method and the one below.